renderer/components/measurement/ExplorerURLButton.js

```diff
@@ -10,7 +10,7 @@ const ExplorerURLButton = ({ reportID, input }) => {
     explorerURL = `${explorerURL}?input=${input}`
   }
   return (
-    <Button onClick={(e) => openInBrowser(explorerURL, e)}>
+    <Button onClick={(e) => openInBrowser(explorerURL, e)} data-testid='button-show-in-explorer' href={explorerURL}>
```

Comment:
Why do we need the `href` attribute here?

Reply:
I needed to find a way to get access to the generated link to browse in explorer. I'm not sure if this a good way though.